id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoDraftServiceImpl.java

```diff
@@ -44,6 +44,8 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.*;
```

Comment:
As per our coding practices, we would like to keep the individual imports, most of the case we would to avoid the usage of *

Reply:
yes corrected.